drivers/gpio/gpio_ht16k33.c

```diff
@@ -36,14 +36,14 @@ struct gpio_ht16k33_data {
 };
 
 static int gpio_ht16k33_cfg(struct device *dev, int access_op,
-			       u32_t pin, int flags)
+			    u32_t pin, int flags)
 {
 	ARG_UNUSED(dev);
 	ARG_UNUSED(access_op);
 	ARG_UNUSED(pin);
 
 	/* Keyscan is input-only */
-	if ((flags & GPIO_DIR_MASK) != GPIO_DIR_IN) {
+	if ((flags & GPIO_INPUT) != GPIO_INPUT) {
```

Comment:
This looks a little weird.  I assume this device only supports input, no output; and that it doesn't support disconnecting the input.  If so, I think this should be:
```
if (((flags & (GPIO_INPUT | GPIO_OUTPUT)) == GPIO_DISCONNECTED)
    || ((flags & GPIO_OUTPUT) != 0)) {
    return -ENOTSUP;
}
```

Reply:
@pabigot Thanks, I have changed it accordingly.